src/model/semantics.ml

```diff
@@ -195,24 +246,14 @@ and nestable_block_elements status elements =
 let tag :
     location:Location.span ->
     status ->
-    Odoc_parser.Ast.tag ->
+    Ast.external_tag ->
     ( Comment.block_element with_location,
-      Odoc_parser.Ast.block_element with_location )
+      internal_tags_removed with_location )
     Result.result =
  fun ~location status tag ->
   let ok t = Result.Ok (Location.at location (`Tag t)) in
   match tag with
-  | (`Author _ | `Since _ | `Version _ | `Inline | `Open | `Closed) as tag ->
-      ok tag
-  | `Canonical { value = s; location = r_location } -> (
-      let path = Reference.read_path_longident r_location s in
-      match path with
-      | Result.Ok path -> ok (`Canonical path)
-      | Result.Error e ->
-          Error.warning status.warnings e;
-          let placeholder = [ `Word "@canonical"; `Space " "; `Code_span s ] in
-          let placeholder = List.map (Location.at location) placeholder in
-          Error (Location.at location (`Paragraph placeholder)))
```

Comment:
I've removed this placeholder logic. This tag isn't intended to be rendered. If it contains a syntax error, only the programmer need to be informed (through a warning) and not users reading the documentation.